internal/app/subsystems/aio/store/postgres/postgres.go

```diff
@@ -20,13 +20,28 @@ import (
 	"github.com/resonatehq/resonate/pkg/promise"
 	"github.com/resonatehq/resonate/pkg/schedule"
 	"github.com/resonatehq/resonate/pkg/subscription"
+	"github.com/resonatehq/resonate/pkg/task"
 	"github.com/resonatehq/resonate/pkg/timeout"
 
 	_ "github.com/lib/pq"
 )
 
 const (
 	CREATE_TABLE_STATEMENT = `
+	CREATE TABLE IF NOT EXISTS tasks (
```

Comment:
Do you mind reordering a bit here? I'm thinking the ordering could be
- promises
- schedules

and then I'm easy, but whatever ordering we go with let's try to be consistent.